### DISCUSSION TITLE
Having an issue pull and push in github?

### DISCUSSION THREAD

**Discussion by juztinlazaro** [community/community#24206] [New to GitHub]
<p>ssh -T <a href="mailto:git@github.com">git@github.com</a><br>
Received disconnect from 13.250.177.223 port 22:11: Bye Bye<br>
Disconnected from 13.250.177.223 port 22</p>

**Comment by chaoticbob** [comment]
<p>Also having this issue. Started out of the blue yesterday.</p>

**Comment by juztinlazaro** [comment]
<p><a class="mention" href="/chaoticbob">@chaoticbob</a> me started early morning</p>

**Comment by chaoticbob** [comment, marked as answer]
<p>I managed to solve this…but I have no idea what changed. Anyways, here’s what I did…and this is Windows 10:</p>
<p>1) Create ~/.ssh/config</p>
<p>2) Added the following</p>
<pre><code>Host github.com
KexAlgorithms=ecdh-sha2-nistp521
</code></pre>
<p>I am now able to pull and push.</p>

**Comment by juztinlazaro** [comment]
<p><a class="mention" href="/chaoticbob">@chaoticbob</a> thanks, it works.</p>

**Comment by juztinlazaro** [comment]
<p>Having an issue again.</p>

**Comment by brianamarie** [comment]
<p>Hi <a class="mention" href="/juztinlazaro">@juztinlazaro</a>! Unfortunately, I don’t know how to solve this issue with pushing and pulling. If you are continuing to have problems, I would recommend contacting <a href="https://github.com/contact" rel="nofollow noopener">GitHub Support</a> directly. They’ll be able to help you troubleshoot with your specific scenario. Best of luck! </p>

**Comment by marzbrie92** [comment]
<p>What should I do to get the same thing… Windows 10 for Andriot i Huawei Nova4e?.. Please teach</p>

**Comment by marzbrie92** [comment]
<p>What should I do to get the same thing… Windows 10 for Andriot i Huawei Nova4e?.. Please teach</p>
